roboslack-api/src/main/java/com/palantir/roboslack/api/attachments/Attachment.java

```diff
@@ -72,7 +72,6 @@ public static Builder builder() {
     @Value.Check
     protected final void check() {
         checkArgument(!Strings.isNullOrEmpty(fallback()), "Attachment fallback message cannot be null or empty");
-        MorePreconditions.checkDoesNotContainMarkdown(FALLBACK_FIELD, fallback());
```

Comment:
Problem here is that even if you have an `Attachment` with `fallback` text of something like:
`Meet me at 12-34 Lane St` or something like `Around ~2pm` it would be marked as invalid